Watchman.Discord/Areas/Initialization/Services/ResponsesInitService.cs

```diff
@@ -22,8 +22,13 @@ public ResponsesInitService(ICommandBus commandBus, ResponsesGetterService respo
 
         public async Task InitNewResponsesFromResources()
         {
-            var responsesInBase = this._responsesGetterService.GetResponsesFromBase();
             var defaultResponses = this._responsesGetterService.GetResponsesFromResources();
+            var responsesInBase = this._responsesGetterService.GetResponsesFromBase().Select(response =>
+            {
+                // Update every response in the DB with availableVariables from defaultResponses
+                return new Response(response.OnEvent, response.Message, response.ServerId, 
```

Comment:
1. przenieś wynik tego First do zmiennej pomocniczej
2. co jeśli dana odpowiedź istnieje już tylko w bazie a z defaultResponses jest już usunięta?
lepiej zrobić z FirstOrDefault

Reply:
po co {} i return w jednolinijkowym linq?

Reply:
to zaraz będzie outdated